<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1961</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="protocols">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="ZIgBeeHA">
     <attribute name="title">
      <string>ZIgBee HA</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_zigbee_webcrwal">
      <property name="geometry">
       <rect>
        <x>242</x>
        <y>20</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>웹에서 가져오기</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>209</width>
        <height>94</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_zigbee_channel">
           <property name="text">
            <string>채널</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_zigbee_channel"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_zigbee_id">
           <property name="text">
            <string>검증 대상 ID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_zigbee_id"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>포트</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_port"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="BLE">
     <attribute name="title">
      <string>BLE</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="BLE_MESH">
     <attribute name="title">
      <string>BLE MESH</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_cmd_gen">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>401</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>새굴림</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>명령 생성</string>
    </property>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="group_level">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>371</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Level</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>새굴림</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>명령어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_level_commands">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>108</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>새굴림</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>횟수</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_count_level">
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>65534</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>371</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>새굴림</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QVBoxLayout" name="vlayout_level_widget"/>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_color">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>371</width>
       <height>221</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Color</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>새굴림</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>명령어</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_color_commands">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>30</y>
        <width>108</width>
        <height>32</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>새굴림</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>횟수</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_count_color">
         <property name="font">
          <font>
           <family>새굴림</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>65534</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>371</width>
        <height>161</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>새굴림</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QVBoxLayout" name="vlayout_color_widget"/>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>371</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="hlayout">
      <item>
       <widget class="QRadioButton" name="radioButton_on">
        <property name="text">
         <string>On</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_off">
        <property name="text">
         <string>Off</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_toggle">
        <property name="text">
         <string>Toggle</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>새굴림</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>횟수</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_count_onoff">
          <property name="font">
           <font>
            <family>새굴림</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>65534</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>568</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>620</y>
       <width>341</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_cmd_gen">
        <property name="font">
         <font>
          <family>새굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>명령 등록</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>580</y>
       <width>242</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>새굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>전체 반복 횟수</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_iter_entire">
        <property name="font">
         <font>
          <family>새굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65534</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>361</width>
       <height>21</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QCheckBox" name="checkBox_connection">
        <property name="font">
         <font>
          <family>새굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>connect</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_disconnection">
        <property name="font">
         <font>
          <family>새굴림</family>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>disconnect</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>361</width>
      <height>831</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>명령 목록</string>
    </property>
    <widget class="QListWidget" name="list_gen_cmd">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>361</width>
       <height>641</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>새굴림</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropOverwriteMode">
      <bool>false</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="movement">
      <enum>QListView::Free</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>670</y>
       <width>351</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_save_cmd">
        <property name="text">
         <string>명령 저장</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_load_cmd">
        <property name="text">
         <string>명령 불러오기</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start">
        <property name="text">
         <string>검증 시작</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>1051</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>진행 상황</string>
    </property>
    <widget class="QTableWidget" name="table_current_result">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>1051</width>
       <height>311</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>클러스터</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>명령</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ATTRIBUTE</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>응답 결과</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>PAYLOADS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>전환 시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>대기 시간</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>판단 결과</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_result">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>360</y>
      <width>1071</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>최종 결과</string>
    </property>
    <widget class="QScrollArea" name="scrollArea_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1041</width>
       <height>381</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1039</width>
        <height>379</height>
       </rect>
      </property>
      <widget class="QListWidget" name="list_result">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1031</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>651</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_18">
      <item>
       <widget class="QPushButton" name="btn_result_reset">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>초기화</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_result_print">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>출력</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_result_save">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>결과 저장</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8"/>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1961</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
